cms/templates/js/course-outline.underscore

```diff
@@ -70,7 +70,7 @@ if (is_proctored_exam) {
 <li class="outline-item outline-<%- xblockType %> <%- visibilityClass %> is-draggable <%- includesChildren ? 'is-collapsible' : '' %> <%- isCollapsed ? 'is-collapsed' : '' %>"
     data-parent="<%- parentInfo.get('id') %>" data-locator="<%- xblockInfo.get('id') %>">
 
-    <span class="draggable-drop-indicator draggable-drop-indicator-before"><i class="icon fa fa-caret-right"></i></span>
+    <span class="draggable-drop-indicator draggable-drop-indicator-before"><span class="icon fa fa-caret-right" aria-hidden="true"></span></span>
```

Comment:
Looking at this further, it's kind of a mess and isn't keyboard friendly. The activator for the accordion is the H3, but it's not focusable. So while it has a `title`, because it isn't focusable, it isn't announced. Since screen readers largely rely on keyboard functionality, and since these aren't really keyboard accessible, I'm hesitant to include sr context because it's pointless. Just make sure the icons are aria-hidden.

We should be using `button`s for these - among other things - but I realize that's far out of scope for this PR.